src/server/data.rs

```diff
@@ -1,24 +1,35 @@
+use chrono::Utc;
 use hyper::StatusCode;
 use semaphore::protocol::Proof;
 use semaphore::Field;
 use serde::{Deserialize, Serialize};
 
-use crate::identity_tree::{
-    Hash, InclusionProof, ProcessedStatus, RootItem, Status, UnprocessedStatus,
-};
+use crate::identity_tree::{Hash, InclusionProof, RootItem};
 use crate::prover::{ProverConfig, ProverType};
 
 #[derive(Debug, Serialize, Deserialize)]
-#[serde(transparent)]
-pub struct InclusionProofResponse(pub InclusionProof);
+pub struct InclusionProofResponse {
```

Comment:
These structs are exactly the same as the ones in `crate::identity_tree`, why not just keep them as wrappers with `#[serde(transparent)]`?

Reply:
I personally don't like mixing the application layers and exposing other modules as part of the module's API. This way there is no need to change code for users of the server module when InclusionProof in identity_tree will change. If you don't find it useful and wan't to keep repo smaller let me know. I will flatten it then. :)

Reply:
Let's keep things simple, let's reuse the structs from `crate::identity_tree`

Reply:
Same for other structs

Reply:
done